### PR TITLE
Add GitHub link

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

File: assets/sass/patterns/molecules/social-links.scss [hunk 1]
```diff
@@ -26,11 +26,40 @@
...
+    fill: $color-text;
+  }
+  &:hover svg path {
+    fill: $color-primary;
+  }
+
+  .github-link--text {
+    @include margin(2 0 0 7);
+
+    &:hover {
```
Comment (thewilkybarkid) [inline assets/sass/patterns/molecules/social-links.scss]: Think this is meant to apply to the parent?

Comment (discodavey) [reply assets/sass/patterns/molecules/social-links.scss]: Have added parent class

**Comment by thewilkybarkid** [comment]
Ah, that's what I was referring to. Should have been clearer.

**Comment by discodavey** [comment]
GitHub text now changes colour when you hover over the logo.

**Comment by davidcmoulton** [comment]
At some viewport widths the link is much wider than the visual indication of the link:

screenshot 2018-11-09 16 44 50

File: source/_patterns/01-molecules/components/social-links.mustache [hunk 2]
```diff
@@ -45,3 +45,18 @@
     </li>
   </ul>
 </div>
+
+{{! GitHub Link }}
```
Comment (davidcmoulton) [inline source/_patterns/01-molecules/components/social-links.mustache]: Remove comment and close up whitespace.

File: source/_patterns/01-molecules/components/social-links.mustache [hunk 3]
```diff
@@ -45,3 +45,18 @@
...
+
+<div class="github-link">
```
Comment (davidcmoulton) [inline source/_patterns/01-molecules/components/social-links.mustache]: Prefer class name `github-link-wrapper`.

File: source/_patterns/01-molecules/components/social-links.mustache [hunk 4]
```diff
@@ -45,3 +45,18 @@
...
+  <a href="https://github.com/elifesciences" class="github-link__item">
```
Comment (davidcmoulton) [inline source/_patterns/01-molecules/components/social-links.mustache]: Prefer class name `github-link`.

File: source/_patterns/01-molecules/components/social-links.mustache (hunk 3 above)
Comment (discodavey) [reply source/_patterns/01-molecules/components/social-links.mustache]: Completed

File: source/_patterns/01-molecules/components/social-links.mustache (hunk 4 above)
Comment (discodavey) [reply source/_patterns/01-molecules/components/social-links.mustache]: Completed

**Pull request merged**